pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -155,15 +151,11 @@ func TestPipelineRunSpec_Invalidate(t *testing.T) {
 		spec    v1beta1.PipelineRunSpec
 		wantErr *apis.FieldError
 	}{{
-		name:    "Empty pipelineSpec",
-		spec:    v1beta1.PipelineRunSpec{},
-		wantErr: apis.ErrMissingField("spec"),
-	}, {
```

Comment:
is this test case covered elsewhere? or is it unnecessary?

Reply:
It is unnecessary indeed (I removed that check : https://github.com/tektoncd/pipeline/pull/3279#discussion_r493597819)